scripts/cibuild

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+if [[ -n "${GIT_COMMIT}" ]]; then
+    GIT_COMMIT="${GIT_COMMIT:0:7}"
+else
+    GIT_COMMIT="$(git rev-parse --short HEAD)"
+fi
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Build application for staging or release.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+        docker-compose build gtsite-service gtsite-assets
+        make clean
+
+        echo "Building static asset bundle..."
+        docker-compose run --rm gtsite-assets
```

Comment:
Same here. I don't think that we should execute this step until after `test` is successful.